### ISSUE TITLE
Legends and JAWS instructions improvement

### ISSUE THREAD

**Issue by mae-financial-com** [highcharts/highcharts#15890]
# Description of the feature
When we are using a JAWS screen reader to navigate through charts and reach the first legend we hear this "Toggle Series visibility region, Toggle Analyst Estimates EPS Button. To toggle button press spacebar".

This is useful but a blind user never knows if there are more legends in this region and the user presses Tab key and goes out of this region missing rest of legends.

Legend buttons act like a group of radio buttons and would be helpful if more information can be provided to the screen reader.

Can something be done like bootstrap radio buttons where the user will know that there are more buttons in this region and he can use the arrow buttons to navigate to them?

# Library related to the feature
Highcharts 

# Proof of Concept/Live example for the feature
Details posted on Highcharts forum - [Highcharts Forum link](https://www.highcharts.com/forum/viewtopic.php?f=9&t=46728)
----

**You can vote for this feature by adding a thumbs-up reaction to this post.**

**Comment by oysteinmoseng** [comment]
Hi, in the latest version these are actually a list of toggle buttons, and should be announced as such. Are you using the latest version of Highcharts?

**Comment by mae-financial-com** [comment]
> Hi, in the latest version these are actually a list of toggle buttons, and should be announced as such. Are you using the latest version of Highcharts?

Thanks a lot. Upgraded to the latest version of highcharts and now it works as expected and JAWS is giving good readings. 👍
